marklogic-data-hub-central/ui/e2e/cypress/support/pages/modeling.tsx

```diff
@@ -6,11 +6,11 @@ class ModelingPage {
   }
 
   newEntityName(str: string) {
-    cy.get('[data-testid=name]').type(str);
+    cy.get('#name').type(str);
```

Comment:
This is a welcome change. Thumbs up!. For actual UI test, the page objects could use the id or other css, xpath locators to access an element and we should not have to add a testid.